drawing_7.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<head>
+	<html lang='en'>
+		<title>drawing 7</title>
+			<meta charset="UTF-8">
+				<link rel="stylesheet" type="text/css" href="drawing_7.css">
```

Comment:
Please fix up the tabs here. Code should always be neat and presentable, and there should be an emphasis on the proper use of tabs.